### PR TITLE
Faker.js data of user, posts, comments

### PR DESCRIPTION
**Pull request by steven-yanh**
**Status**

Not Ready

**Description**

Please include a short summary.

**Change Logs**

### CODE REVIEW

File: seeders/20221018204029-seeding-post.js [hunk 1]
```diff
@@ -1,16 +1,121 @@
...
+
+async function createUser() {
+  for (let i = 1; i <= 1000; i++) {
+    const user = {
+      email: faker.internet.email(),
+      phone: faker.phone.number("###-###-####"),
+      username: faker.internet.userName(),
+      password: await User.encryptPassword(faker.internet.password()),
+      role: "user",
+      last_edit_date: faker.date.between(
```
Comment (LandingHeart) [inline seeders/20221018204029-seeding-post.js:18]: lets remove the last edit date, create a migration file to remove that as it is not going to be used.

File: seeders/20221018204029-seeding-post.js [hunk 2]
```diff
@@ -1,16 +1,121 @@
...
+  ];
+  for (let userId = 1; userId <= 1000; userId++) {
+    //generate 0-3 post for each user
+    const numberOfPost = faker.datatype.number({ min: 0, max: 3 });
+    for (let i = 0; i < numberOfPost; i++) {
+      const aspectRatio = faker.helpers.arrayElement(aspectRatios);
+      const width = aspectRatio[0];
+      const height = aspectRatio[1];
+
+      posts.push({
```
Comment (LandingHeart) [inline seeders/20221018204029-seeding-post.js:49]: have you figure how to store the image generated and firebase?

Comment (steven-yanh) [reply seeders/20221018204029-seeding-post.js:49]: I haven't set up the Firebase in the web app but these changes are like a checkpoint so for the next step I will try to upload these generated images to Firebase.

Comment (LandingHeart) [reply seeders/20221018204029-seeding-post.js:64]: Okay, just dont run the seed yet if merged to prod.

**Review by LandingHeart — APPROVED** [review]

**Pull request merged**